app/api/models/errors/MtdErrors.scala

```diff
@@ -131,6 +131,12 @@ object RuleDateRangeInvalidError
       message = "The specified date range is invalid"
     )
 
+object RuleInconsistentQueryParamsError
+  extends MtdError(
+    code = "RULE_INCONSISTENT_QUERY_PARAMS",
+    message = "Provide date range or docNumber when onlyOpen items are false"
```

Comment:
Appreciate this message is just copied from the spec, but it'd be better as:

```suggestion
    message = "Provide date range or docNumber when onlyOpenItems is false."
```